### PR TITLE
Rmsharp branch

### PR DESCRIPTION
**Pull request by rmsharp**
This was the package I initially was trying to use that has a dependency on checkr.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,46 +1,12 @@
...
-    template:
-    - "%{repository}#%{build_number} (%{branch} - %{commit} : %{author}): %{message}
-      | Details: %{build_url} | Changes: %{compare_url}"
-    rooms:
-      secure: SQirvWbQ9b0roApmf3gt6JTcWIra9NguGzR45azxVDaWw2n0w/sIufA/cxa2sTLLhKfIMNlJwwhQjNaWyHeZkTTxRb76tcHWQdPpMcNKTwfr3+C3/bXDkmQozvQkYNgGuRc2Iln5ms12fiHmwI6dp5aENACUo5fbV2SLJQvmt5w=
+  - Rscript -e 'covr::codecov()'
+sudo: false
+os:
+  - osx
+warnings_are_errors: true
```
Comment (peterhurford) [inline .travis.yml]: This change to Travis removes the testing. I made a PR here https://github.com/peterhurford/strdate/pull/6 to fix Travis for this package. Can you use that `.travis.yml` file instead?

**Comment by peterhurford** [comment]
Thanks again for your contributions! 👍  We can eventually get this package on CRAN soon as well (after checkr). 👍 

The only change I need is for you to use my updated `.travis.yml` file (and then ensure Travis passes).

**Comment by rmsharp** [comment]
I did not see this prior to my PR I just sent. I had gone to sleep with Travis running and did not want to send the change unless the Travis build was clean. Build #3 is. See it at https://travis-ci.org/rmsharp/strdate/

Why do you want to turn off testing?

Mark
R. Mark Sharp
7526 Meadow Green St.
San Antonio, TX 78251
mobile: 210-218-2868
rmsharp@me.com

**Comment by peterhurford** [comment]
@rmsharp Your changes to the Travis file remove all the automated testing, so a green build is meaningless. This is not something that I can accept. I am happy to accept your PR, but I need it to run with my chosen test suite. If you like, I can integrate and work on it for you.

**Comment by rmsharp** [comment]
Sorry, I did not realize I turned off testing.

Mark
R. Mark Sharp
7526 Meadow Green St.
San Antonio, TX 78251
mobile: 210-218-2868
rmsharp@me.com

**Comment by rmsharp** [comment]
On #6 of https://travis-ci.org/rmsharp/strdate, checkr loaded clean, but it is still reporting these two items:

checking dependencies in R code
   ':::' call which should be '::': ‘checkr:::validate_’
     See the note in ?`:::` about the use of this operator.

N  checking R code for possible problems (1.4s)
   date_range: no visible global function definition for ‘head’
   strdate: no visible global function definition for ‘head’
   Undefined global functions or variables:
     head
   Consider adding
     importFrom("utils", "head")
   to your NAMESPACE file.

R. Mark Sharp
7526 Meadow Green St.
San Antonio, TX 78251
mobile: 210-218-2868
rmsharp@me.com

**Comment by peterhurford** [comment]
@rmsharp We can clean that up, no worries. For now I'm going to merge this! 👍

**Comment by rmsharp** [comment]
Thanks.

I deleted my local fork and merged with your version again. Some how it still did not get the namespace corrections for those last two we discussed. I am guessing you are working on other things. 

I will check back tomorrow. I am trying to use your two packages as dependencies of testthatsomemore, which I would like for some unit testing of a package (nprcmanager) I am working on.

From https://travis-ci.org/rmsharp/strdate/builds/286745607


N  checking dependencies in R code
   ':::' call which should be '::': ‘checkr:::validate_’
     See the note in ?`:::` about the use of this operator.

and 

N  checking R code for possible problems (1.5s)
   date_range: no visible global function definition for ‘head’
   strdate: no visible global function definition for ‘head’
   Undefined global functions or variables:
     head
   Consider adding
     importFrom("utils", "head")
   to your NAMESPACE file.



R. Mark Sharp
7526 Meadow Green St.
San Antonio, TX 78251
mobile: 210-218-2868
rmsharp@me.com

**Pull request merged**